core/embed/firmware/memory_T3T1.ld

```diff
@@ -110,6 +105,11 @@ SECTIONS {
     . = ALIGN(4);
   } >SRAM3
 
+  .buf : ALIGN(4) {
+    *(.buf*);
+    . = ALIGN(4);
+  } >SRAM3
+
```

Comment:
I increased the size of ImageBuffer so that it can fit the whole 240x240px jpg - which will be needed for animating homescreen. It would not fit the SRAM1, so i moved the smaller stuff in `.buf` to SRAM3, to achieve smaller effect on micropython heap size than moving the image buffer there.